elasticdl/python/tests/pserver_servicer_test.py

```diff
@@ -469,6 +469,63 @@ def test_save_parameters_to_checkpoint_file(self):
                 ["variables-0-of-1.chkpt"],
             )
 
+    def test_restore_parameters_from_checkpoint(self):
+        checkpoint_dir_for_init = (
+            "elasticdl/python/tests/testdata/functional_ckpt/version-100"
```

Comment:
The test data directory for `functional_ckpt/version-100` is not included in this PR